packages/node_modules/webex/src/webex.js

```diff
@@ -23,32 +23,31 @@ require('@webex/plugin-team-memberships');
 require('@webex/plugin-webhooks');
 
 const merge = require('lodash/merge');
-const SparkCore = require('@webex/webex-core').default;
-
+const WebexCore = require('@webex/webex-core').default;
 const config = require('./config');
 
 // documentation.js puts hashes in relative urls, so need to specify full urls
 // here
 /**
- * See {@link https://webex.github.io/spark-js-sdk/example/browsers|Browser Guide} and
- * {@link https://webex.github.io/spark-js-sdk/example/servers|Server Guide}
+ * See {@link https://webex.github.io/webex-js-sdk/example/browsers|Browser Guide} and
+ * {@link https://webex.github.io/webex-js-sdk/example/servers|Server Guide}
  * @see {@link /example/browsers|Browser Guide}
  * @see {@link /example/servers|Server Guide}
- * @class CiscoSpark
+ * @class Webex
  */
-const CiscoSpark = SparkCore.extend({
-  ciscospark: true,
+const Webex = WebexCore.extend({
+  webex: true,
```

Comment:
This is a property for telemetry. Can be changed and updated on whomever's side is doing the analytics.